src/test/java/programminglife/parser/GraphParserTest.java

```diff
@@ -122,4 +137,112 @@ public void update(Observable o, Object arg) {
             }
         }
     }
+
+    @Test
+    public void parseSegmentNotStartingS() {
+        try {
+            graphParser.parseSegment("Fiets");
+            fail("This should not be parsed");
+        } catch (Exception e) {
+            assertEquals(UnknownTypeException.class, e.getClass());
+            assertEquals("Line (Fiets) is not a segment", e.getMessage());
+        }
+    }
```

Comment:
The reason for this method is to test the message. It would probably be better to use more specific `Exception`s, but this is a bit overkill for now.